tests/topotests/bgp_lu_topo1/R3/bgpd.conf

```diff
@@ -0,0 +1,523 @@
+  network 11.0.1.134/32
+  network 11.0.1.135/32
+  network 11.0.1.136/32
+  network 11.0.1.137/32
+  network 11.0.1.138/32
+  network 11.0.1.139/32
+  network 11.0.1.140/32
+  network 11.0.1.141/32
+  network 11.0.1.142/32
+  network 11.0.1.143/32
+  network 11.0.1.144/32
+  network 11.0.1.145/32
+  network 11.0.1.146/32
+  network 11.0.1.147/32
+  network 11.0.1.148/32
+  network 11.0.1.149/32
+  network 11.0.1.150/32
+  network 11.0.1.151/32
+  network 11.0.1.152/32
+  network 11.0.1.153/32
+  network 11.0.1.154/32
+  network 11.0.1.155/32
+  network 11.0.1.156/32
+  network 11.0.1.157/32
+  network 11.0.1.158/32
+  network 11.0.1.159/32
+  network 11.0.1.160/32
+  network 11.0.1.161/32
+  network 11.0.1.162/32
+  network 11.0.1.163/32
+  network 11.0.1.164/32
+  network 11.0.1.165/32
+  network 11.0.1.166/32
+  network 11.0.1.167/32
+  network 11.0.1.168/32
+  network 11.0.1.169/32
+  network 11.0.1.170/32
+  network 11.0.1.171/32
+  network 11.0.1.172/32
+  network 11.0.1.173/32
+  network 11.0.1.174/32
+  network 11.0.1.175/32
+  network 11.0.1.176/32
+  network 11.0.1.177/32
+  network 11.0.1.178/32
+  network 11.0.1.179/32
+  network 11.0.1.180/32
+  network 11.0.1.181/32
+  network 11.0.1.182/32
+  network 11.0.1.183/32
+  network 11.0.1.184/32
+  network 11.0.1.185/32
+  network 11.0.1.186/32
+  network 11.0.1.187/32
+  network 11.0.1.188/32
+  network 11.0.1.189/32
+  network 11.0.1.190/32
+  network 11.0.1.191/32
+  network 11.0.1.192/32
+  network 11.0.1.193/32
+  network 11.0.1.194/32
+  network 11.0.1.195/32
+  network 11.0.1.196/32
+  network 11.0.1.197/32
+  network 11.0.1.198/32
+  network 11.0.1.199/32
+  network 11.0.1.200/32
+  network 11.0.1.201/32
+  network 11.0.1.202/32
+  network 11.0.1.203/32
+  network 11.0.1.204/32
+  network 11.0.1.205/32
+  network 11.0.1.206/32
+  network 11.0.1.207/32
+  network 11.0.1.208/32
+  network 11.0.1.209/32
+  network 11.0.1.210/32
+  network 11.0.1.211/32
+  network 11.0.1.212/32
+  network 11.0.1.213/32
+  network 11.0.1.214/32
+  network 11.0.1.215/32
+  network 11.0.1.216/32
+  network 11.0.1.217/32
+  network 11.0.1.218/32
+  network 11.0.1.219/32
+  network 11.0.1.220/32
+  network 11.0.1.221/32
+  network 11.0.1.222/32
+  network 11.0.1.223/32
+  network 11.0.1.224/32
+  network 11.0.1.225/32
+  network 11.0.1.226/32
+  network 11.0.1.227/32
+  network 11.0.1.228/32
+  network 11.0.1.229/32
+  network 11.0.1.230/32
+  network 11.0.1.231/32
+  network 11.0.1.232/32
+  network 11.0.1.233/32
+  network 11.0.1.234/32
+  network 11.0.1.235/32
+  network 11.0.1.236/32
+  network 11.0.1.237/32
+  network 11.0.1.238/32
+  network 11.0.1.239/32
+  network 11.0.1.240/32
+  network 11.0.1.241/32
+  network 11.0.1.242/32
+  network 11.0.1.243/32
+  network 11.0.1.244/32
+  network 11.0.1.245/32
+  network 11.0.1.246/32
+  network 11.0.1.247/32
+  network 11.0.1.248/32
+  network 11.0.1.249/32
+  network 11.0.1.250/32
+  network 11.0.1.251/32
+  network 11.0.1.252/32
+  network 11.0.1.253/32
```

Comment:
non-blocking but why not have sharpd install the routes and redistribute sharpd?  Would make this config much much smaller